UI/update readme

eao10000 opened the pull request:
@xWngl and I have finished cleaning up and updating the READMME.md in the ui directory. We added additional information about the coordinates module while also removing the DFS module from the README.md as it was no longer needed. We were also able to update the how to use section making it easier and more accurate on how to run the ui directory.

xWngl reviewed (approved):
I'd just add that the old DFS module is now part of the coordinates module

MaxineK36 commented on src/ui/README.md:
```diff
@@ -11,10 +11,11 @@ are a few currently enabled commands.
+files can be found in the src/ui/examples subdirectory from build. You can create
+the binaries for all of these files by running "make" inside the examples 
+subdirectory or using "make file_name" for a specific example. Use "./file_name"
+to run the executable.
```
Can we also specify what examples are currently there?

MaxineK36 reviewed (changes requested):
Are the `window` and `map` modules still being used? If not, we should note that. Otherwise, looks good modulo my one comment!

MaxineK36 reviewed (approved):
LGTM!

eao10000 commented:
The README.md in the UI module has been updated and implemented. When doing this we noticed that the coordinates section was not completed, so we added the information there, while deleting the unused DFS section. With this the issue is closed.

MaxineK36 commented:
Issue Score: ✔️++

Comments:
Great work! Going forward, please make sure that everyone who contributed to the code is an assignee on the PR, and that they are not also assigned as reviewers.

<!-- {"score": "check-plus-plus"} -->
